schema.sql

```diff
@@ -0,0 +1,88 @@
+DROP TABLE IF EXISTS application;
+DROP TYPE IF EXISTS application_status;
+DROP TABLE IF EXISTS applicant_phone_number;
+DROP TYPE IF EXISTS phone_type;
+DROP TABLE IF EXISTS applicant_address;
+DROP TABLE IF EXISTS applicant;
+DROP TABLE IF EXISTS advisor;
+DROP TYPE IF EXISTS role;
+DROP TABLE IF EXISTS "user";
+
+CREATE TABLE "user"
```

Comment:
should be great add "if not exists" for all create tables

Reply:
Done